blog/2015/05/online-instructor-training.html

```diff
@@ -0,0 +1,53 @@
+---
+layout: blog
+root: ../../..
+author: Greg Wilson
+title: "Online Instructor Training Revisited"
+date: 2015-05-18
+time: "17:00:00"
+category: ["Instructor Training"]
+---
+<!-- start excerpt -->
+<p>
+  We have now run instructor training in three formats:
+  an in-person two- or three-day class,
+  a multi-week online class,
+  and a hybrid version in which the trainees are co-located,
+  but the trainer comes in via the web.
+  We've also gathered
+  <a href="https://github.com/swcarpentry/training-course/issues/514">a lot of feedback</a>
+  on what people want from instructor training
+  and <a href="https://github.com/swcarpentry/training-course/issues/445">what its prerequisites should be</a>.
+  Based on all of that,
+  we're going to try to combine the best features of everything we've done so far.
+</p>
+<!-- end excerpt -->
+<p>
+  On June 8-9,
+  we're going to run instructor training for 18 people at three sites:
+  one in South Africa,
+  one in the United Kingdom,
+  and one in the United States.
+  Three days later,
+  on June 12-13,
+  we're going to do it again for about 15 people at another three sites
+  that belong to one of our partner organizations.
+  Myself,
+  Aleksandra Pawlik from the <a href="http://software.ac.uk/">Software Sustainability Institute</a>,
+  and Warren Code from the <a href="http://sclt.science.ubc.ca/">UBC Science Centre for Learning and Teaching</a>
+  will give these trainees the same lessons and exercises as participants in the live course;
+  they will then do the same graduation exercise as our online trainees
+  (i.e., submit a change to one of our lessons and teach online for five minutes).
+</p>
+<p>
+  If this goes well,
+  we will then decide what to do about instructor training in future.
+  People enjoy the multi-week course,
+  but it's been a growing drain on scarce resources.
```

Comment:
...and for some participants it is difficult to stick out with the multi-week course, so the compact online course could be a good alternative for them.